### PR TITLE
Refactoring and PEP8 fixes.

### PR DESCRIPTION
**Pull request by OlafSzmidt**
Some code PEP8 fixes as well as refactoring when I was reading the codebase trying to get hang of it ;)

### CODE REVIEW

File: aimmo-game/simulation/avatar/fog_of_war.py [hunk 1]
```diff
@@ -36,6 +36,7 @@ def apply_fog_of_war(world_map, avatar_wrapper):
 
 
 def should_partially_fog(no_fog_distance, partial_fog_distance, x_dist, y_dist):
+    # TODO: partial_fog_distance implemented in this method somehow.
```
Comment (CelineBoudier) [inline aimmo-game/simulation/avatar/fog_of_war.py]: we tend not to commit todo comments

Comment (CelineBoudier) [reply aimmo-game/simulation/avatar/fog_of_war.py]: I'd rather we open an issue

File: aimmo-game/simulation/turn_manager.py [hunk 2]
```diff
@@ -61,15 +62,16 @@ def _register_action(self, avatar):
...
 
-    def _update_environment(self, game_state):
+    @staticmethod
+    def _update_environment(game_state):
         num_avatars = len(game_state.avatar_manager.active_avatars)
         game_state.world_map.reconstruct_interactive_state(num_avatars)
 
     def _mark_complete(self):
         pass
         # TODO: Make completion request work. For now, we assume games don't finish.
```
Comment (CelineBoudier) [inline aimmo-game/simulation/turn_manager.py]: same here

File: aimmo-game/simulation/avatar/fog_of_war.py (hunk 1 above)
Comment (OlafSzmidt) [reply aimmo-game/simulation/avatar/fog_of_war.py]: #240 opened.

File: aimmo-game/simulation/turn_manager.py (hunk 2 above)
Comment (OlafSzmidt) [reply aimmo-game/simulation/turn_manager.py]: #241 opened.

**Comment by OlafSzmidt** [comment]
@CelineBoudier Do you want me to recommit the files without the comments, or leave them with the issues?

**Comment by CelineBoudier** [comment]
@OlafSzmidt please recommit without the comments :) thanks

File: aimmo-game/simulation/turn_manager.py [hunk 3]
```diff
@@ -61,15 +62,15 @@ def _register_action(self, avatar):
...
         num_avatars = len(game_state.avatar_manager.active_avatars)
         game_state.world_map.reconstruct_interactive_state(num_avatars)
 
     def _mark_complete(self):
         pass
-        # TODO: Make completion request work. For now, we assume games don't finish.
-        #from world_state import WorldState
-        #requests.post(self._completion_url, json=world_state.get_update())
+        # from world_state import WorldState
+        # requests.post(self._completion_url, json=world_state.get_update())
```
Comment (CelineBoudier) [inline aimmo-game/simulation/turn_manager.py]: we don't commit commented code either :)

File: aimmo-game/simulation/world_map.py [hunk 4]
```diff
@@ -208,7 +209,7 @@ def _update_map(self, num_avatars):
         self._add_pickups(num_avatars)
 
     def _expand(self, num_avatars):
-        #LOGGER.info('Expanding map')
+        # LOGGER.info('Expanding map')
```
Comment (CelineBoudier) [inline aimmo-game/simulation/world_map.py]: same

**Comment by OlafSzmidt** [comment]
@CelineBoudier Should be done now. :)) Github doesn't recognise squashed commits.

**Pull request merged**